README.md

```diff
@@ -54,6 +54,10 @@ The easiest way to get an Algo server running is to run it on your local system
             ```bash
             sudo apt install -y --no-install-recommends python3-virtualenv
             ```
+            * RPi ( ubuntu server: install these packages in conjunction with above dependencies ):
+                ```
+                sudo apt install libffi-dev libssl-dev
+                ```
```

Comment:
I think this would be clearer and more consistent:
```suggestion
            On a Raspberry Pi running Ubuntu also install `libffi-dev` and `libssl-dev`.
```

Reply:
no problem, it is now added :grin:

thanks for the suggestion :pray:, and that does make it flow better :slightly_smiling_face: